runtime/IO/File.cpp

```diff
@@ -92,4 +92,11 @@ bool File::exists(String path)
     return true;
 }
 
+ErrorOr<void> mkdir(String path)
+{
+    if (::mkdir(path.c_string(), S_IRWXU) == -1)
```

Comment:
the 2 argument function does not exist on Windows. Therefore, you can either

```c++
#ifdef _WIN32
   if (::mkdir(path.c_string()) == -1)
#else
  if (::mkdir(path.c_string(), S_IRWXU) == -1)
#endif
```

or reproduce the entire function for each platform, like so:

```
#ifdef _WIN32
ErrorOr<void> mkdir(String path)
{
    if (::mkdir(path.c_string()) == -1) {
        return Error::from_errno(errno);
    }
    return ErrorOr<void> {};
}
#else
ErrorOr<void> mkdir(String path)
{
    if (::mkdir(path.c_string(), S_IRWXU) == -1)
        return Error::from_errno(errno);
    return ErrorOr<void> {};
}
#endif
```